src/components/RefundBankAccount.tsx

```diff
@@ -1,21 +1,25 @@
-import React from 'react'
+import React, { ReactElement } from 'react'
 import { useForm } from 'react-hook-form'
 import { Box } from '@material-ui/core'
 import { useDispatch, useSelector } from 'react-redux'
-import { LabeledInput } from './labeledInput'
+import { LabeledInput } from './input/LabeledInput'
```

Comment:
Could this and some of the other input imports be changed to 

`import { LabeledInput } from './input'`

because of the addition of the input index file?